src/components/modal.jsx

```
@@ -52,66 +52,66 @@
                   </span>
                 ))}
               </div>
               
-              <div className='bar'>
-                <p className='bar-label'>Affection Level</p>
-              <div className='affection--bar-container'>
-                <div className='affection--bar' style={{ '--affectionWidth': `${affectionLevel}%` }}>.</div>
+              <div className='mt-4'>
+                <p className='font-light text-sm'>Affection Level</p>
+              <div className='w-full bg-yellow-200 rounded h-2.5 mt-1'>
+                <div className='bg-yellow-500 h-2.5 rounded' style={{ width: `${affectionLevel}%` }}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)